### ISSUE TITLE
Unable to load the website after generating a layout

### ISSUE THREAD

**Issue by mrwm** [adamws/keyboard-tools#11]
I made a long layout located [in this gist](https://gist.github.com/mrwm/90f5224271c2abcb10bbcefd70cd28ef), and was able to upload it to the website, but it timed out and gave an error saying something along the lines of being unable to receive build status of the project.

<details><summary>Attempting to reload brings up this error in firefox:
</summary>
 <img src="https://github.com/adamws/keyboard-tools/assets/8944799/9013f95a-4bab-4609-9a46-4f5de2d3091e">
</details>

**Comment by mrwm** [comment]
I just tried again, and this error occurs

```
Traceback (most recent call last):
File "/kicad/src/tasks.py", line 71, in generate_kicad_project
log_path = kicad.new_pcb(task_id, task_request, __update_percentage)
File "/kicad/src/kicad.py", line 280, in new_pcb
generate_pcb_file(project_full_path, project_name)
File "/kicad/src/kicad.py", line 122, in generate_pcb_file
raise Exception(f"Generate .kicad_pcb from netlist failed: details: {log}")
Exception: Generate .kicad_pcb from netlist failed: details: ./kicad/pcbnew/action_plugin.cpp(163): assert "PgmOrNull()" failed in register_action().
Unable to find footprint Stabilizer_Cherry_MX_2u in /kicad/e57f1ae3-3e5d-421e-866b-2b0dc980e875/keyboard/libs/keyswitch-kicad-library/footprints/Mounting_Keyboard_Stabilizer.pretty
Unable to find footprint Stabilizer_Cherry_MX_2u in /kicad/e57f1ae3-3e5d-421e-866b-2b0dc980e875/keyboard/libs/keyswitch-kicad-library/footprints/Mounting_Keyboard_Stabilizer.pretty
Unable to find footprint Stabilizer_Cherry_MX_2u in /kicad/e57f1ae3-3e5d-421e-866b-2b0dc980e875/keyboard/libs/keyswitch-kicad-library/footprints/Mounting_Keyboard_Stabilizer.pretty
Unable to find footprint Stabilizer_Cherry_MX_2u in /kicad/e57f1ae3-3e5d-421e-866b-2b0dc980e875/keyboard/libs/keyswitch-kicad-library/footprints/Mounting_Keyboard_Stabilizer.pretty
Unable to find footprint Stabilizer_Cherry_MX_2u in /kicad/e57f1ae3-3e5d-421e-866b-2b0dc980e875/keyboard/libs/keyswitch-kicad-library/footprints/Mounting_Keyboard_Stabilizer.pretty
Unable to find footprint Stabilizer_Cherry_MX_6u in /kicad/e57f1ae3-3e5d-421e-866b-2b0dc980e875/keyboard/libs/keyswitch-kicad-library/footprints/Mounting_Keyboard_Stabilizer.pretty
```

This is with the following options and using the layout in the gist linked above:

- Matrix: Automatic
- Footprints: MX
- Routing: Full
- Controller circuit: None

**Comment by adamws** [comment]
thank you for reporting, based on the log it is related to https://github.com/adamws/keyboard-tools/issues/10 issue
something is definitively buggy, I will have a look in my spare time

**Comment by adamws** [comment]
@mrwm Give it a try now, I just deployed [v0.3](https://github.com/adamws/keyboard-tools/releases/tag/v0.3)

There were two problems you encountered:
- wrong stabilizer footprint name for MX switches (6fa7c8732627f8574aa862ce3927ebff881051ea)
- poor performance. Some big layouts could hog server. I'm running this on 1CPU 1GB RAM machine. I found out that one of the dependencies were consuming a lot of resources - in worst case scenario it could even crash server. It should be better now (21869256386fd4555e5b6f312e03d6d4217956d2)

I've run your layout (from first message) and it succeed.
It **would** fail if you try to use  `Controller circuit: ATmega32U4` option but that is an issue for another day.
